env_run.xml

```diff
@@ -1131,12 +1131,12 @@
         <value compclass="LND">FALSE</value>
       </values>
     </entry>
-    <entry id="DATA_ASSIMILATION_CYCLES" value="4">
+    <entry id="DATA_ASSIMILATION_CYCLES" value="1">
       <type>integer</type>
       <valid_values/>
       <desc> Number of model run - data assimilation steps to complete </desc>
     </entry>
-    <entry id="DATA_ASSIMILATION_SCRIPT" value="/glade/work/raeder/Exp/f.e21.FHIST_BGC.f09_025.CAM6assim.011/assimilate.csh">
+    <entry id="DATA_ASSIMILATION_SCRIPT" value="/glade/work/raeder/Exp/f.e21.FHIST_BGC.f09_025.CAM6assim.011/no_assimilate.csh">
```

Comment:
We are trying to test the old filter here, right? Why are you switching to no_assimilate script?

Reply:
Got it. Then, I am fine with the rest of the changes.

Reply:
I admit I'm a bit confused. I thought the first hindcast was identical, so I don't see the need to hindcast again. I thought you'd just run the assimilation. Not a big expense, so I don't really object.

Reply:
It was identical, but I ran the whole day, so the h6 hindcast state ensemble
was first overwritten by the assimilation, then purged by later cycles.
So I'm recreating the hindcast, and archiving it before letting the h6 assimilation
overwrite it.  It's more complicated than anyone would like.

Reply:
Ah ... at one point you said the hindcasts were identical, so I thought they were available.